array_api_compat/common/_helpers.py

```diff
@@ -439,7 +437,11 @@ def _check_api_version(api_version: str) -> None:
         raise ValueError("Only the 2024.12 version of the array API specification is currently supported")
 
 
-def array_namespace(*xs, api_version=None, use_compat=None) -> Namespace:
+def array_namespace(
+    *xs: Union[Array, bool, int, float, complex, None],
```

Comment:
it's sufficient to write

```suggestion
    *xs: Array | complex | None,
```

Keep in mind that in case of type-checker errors, often the entire signature is printed. So it's usually a good idea to avoid long annotations (in function signatures, but also in general).